svelte.config.mjs

```diff
@@ -31,8 +33,8 @@ const config = {
     },
     adapter: ghpagesAdapter({
       // default options are shown
-      pages: "build",
-      assets: "build",
+      pages: outDir,
+      assets: outDir,
```

Comment:
This is changes the output destination of the documentation build files.
With this change, we can test static site by simply using the `npx http-server` command in `build` directory.